### PR TITLE
 fix the  wrong word

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: promql/engine.go [hunk 1]
```diff
@@ -1350,7 +1350,7 @@ func (ev *evaluator) matrixIterSlice(it *storage.BufferedSeriesIterator, mint, m
 			ev.currentSamples++
 		}
 	}
-	// The seeked sample might also be in the range.
+	// The sought sample might also be in the range.
```
Comment (csmarchbanks) [inline promql/engine.go]: Seeked is a non-standard, but correct past tense of seek, and in these cases easier to read than changing the word to sought.

File: tsdb/querier_test.go [hunk 2]
```diff
@@ -1062,7 +1062,7 @@ func TestChunkSeriesIterator_DoubleSeek(t *testing.T) {
 	testutil.Equals(t, float64(2), v)
 }
 
-// Regression when seeked chunks were still found via binary search and we always
+// Regression when sought chunks were still found via binary search and we always
```
Comment (csmarchbanks) [inline tsdb/querier_test.go]: Same as above, seeked is fine.

File: promql/engine.go (hunk 1 above)
Comment (tanjunchen) [reply promql/engine.go]: OK Thank You

File: tsdb/querier_test.go (hunk 2 above)
Comment (tanjunchen) [reply tsdb/querier_test.go]: OK Thank You

File: promql/engine.go (hunk 1 above)
Comment (tanjunchen) [reply promql/engine.go]: I Changed it Any problem?

**Comment by tanjunchen** [comment]
@csmarchbanks

**Pull request merged**